static/scenes/railroad/js/present.js

```diff
@@ -2,12 +2,18 @@ goog.provide('app.Present');
 
 const material0 = new THREE.MeshToonMaterial( {color: 0xF9D231}); 
 
+// the distance to the target scaled by this number is how high we go
+const midpointOffset = .25;
+
+// a value from 0 to 1 that defines the peak height
+const midpointT = .25;
```

Comment:
Why is this not 0.5? Should the control point for the peak be in the middle of the path?

Reply:
Perhaps a bad name for the variable. Given a path from A->B, this is the point at which the present reaches peak height.

The number .25 was chosen from hand tuning, but I'd like to change the entire lobbing algorithm (specifically the maths you posted before).